core/model/objects/message.hpp

```diff
@@ -18,20 +18,22 @@ limitations under the License.
 #ifndef CORE_DOMAIN_OBJECTS_MESSAGE_HPP_
 #define CORE_DOMAIN_OBJECTS_MESSAGE_HPP_
 
-#include "object.hpp"
-#include "../../service/json_parse.hpp"
 #include <string>
+#include <memory>
 
-namespace message {
+#include "../../service/json_parse.hpp"
 
-class Message : public AbsObject{
-    std::string text;
+namespace object {
 
+class Message{
+
+    std::string text;
 public:
-    Message(std::string text);
+    Message(const std::string& text);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class 'Message' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4478421700/issues/source?bid=3712501&fileBranchId=3923761#l32)